hpx/runtime/parcelset/locality.hpp

```diff
@@ -1,6 +1,7 @@
 //  Copyright (c)      2014 Thomas Heller
 //  Copyright (c) 2007-2017 Hartmut Kaiser
 //  Copyright (c) 2007 Richard D. Guidry Jr.
+//  Copyright 2002, 2005 Daryle Walker
```

Comment:
This looks out of place.

Reply:
Yah, that's not right. Thanks for spotting.